tests/unit/LHNOrderTest.js

```diff
@@ -152,165 +17,131 @@ const ONYXKEYS = {
-                    [`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}1`]: fakeReport1Actions,
+                    [ONYXKEYS.NVP_PRIORITY_MODE]: CONST.PRIORITY_MODE.DEFAULT,
+                    [ONYXKEYS.PERSONAL_DETAILS]: LHNTestUtils.fakePersonalDetails,
+                    [ONYXKEYS.CURRENTLY_VIEWED_REPORTID]: report.reportID.toString(),
+                    [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
                 }))
 
-                // Then the component should be rendered with an item for the fake report
+                // Then the component should be rendered with an item for the report
                 .then(() => {
-                    expect(sidebarLinks.toJSON()).not.toBe(null);
-                    expect(sidebarLinks.toJSON().children.length).toBe(2);
-                    expect(sidebarLinks.getAllByText('ReportID, One')).toHaveLength(1);
+                    expect(sidebarLinks.queryAllByText('One, Two')).toHaveLength(1);
                 });
         });
 
-        test('orders items with most recently updated on top', () => {
-            // Given the sidebar is rendered in default mode (most recent first)
-            // while currently viewing report 1
-            const sidebarLinks = getDefaultRenderedSidebarLinks();
+        it('orders items with most recently updated on top', () => {
+            const sidebarLinks = LHNTestUtils.getDefaultRenderedSidebarLinks();
+
+            // Given three unread reports in the recently updated order of 3, 2, 1
+            const report1 = {
+                ...LHNTestUtils.getFakeReport(['email1@test.com', 'email2@test.com'], 3),
+                lastReadSequenceNumber: LHNTestUtils.TEST_MAX_SEQUENCE_NUMBER - 1,
+            };
+            const report2 = {
+                ...LHNTestUtils.getFakeReport(['email3@test.com', 'email4@test.com'], 2),
+                lastReadSequenceNumber: LHNTestUtils.TEST_MAX_SEQUENCE_NUMBER - 1,
+            };
+            const report3 = {
+                ...LHNTestUtils.getFakeReport(['email5@test.com', 'email6@test.com'], 1),
+                lastReadSequenceNumber: LHNTestUtils.TEST_MAX_SEQUENCE_NUMBER - 1,
+            };
 
             return waitForPromisesToResolve()
 
-                // When Onyx is updated with some personal details and multiple reports
+                // When Onyx is updated with the data and the sidebar re-renders
                 .then(() => Onyx.multiSet({
-                    [ONYXKEYS.NVP_PRIORITY_MODE]: 'default',
-                    [ONYXKEYS.PERSONAL_DETAILS]: fakePersonalDetails,
-                    [ONYXKEYS.CURRENTLY_VIEWED_REPORTID]: '1',
-                    [`${ONYXKEYS.COLLECTION.REPORT}1`]: fakeReport1,
-                    [`${ONYXKEYS.COLLECTION.REPORT}2`]: fakeReport2,
-                    [`${ONYXKEYS.COLLECTION.REPORT}3`]: fakeReport3,
-                    [`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}1`]: fakeReport1Actions,
```

Comment:
I tried to reduce these tests to as little as necessary to still test what we needed to test. The report actions are not used anywhere in the logic, so I just removed them.